packages/ts-transform/src/css-prop/__tests__/index.test.tsx

```diff
@@ -737,9 +741,9 @@ describe('css prop transformer', () => {
         \`}>hello world</div>
       `);
 
-      expect(actual).toInclude('style={{ "--var-test-n30gray": N30 }}');
+      expect(actual).toInclude('<div className="css-test">hello world</div>');
       expect(actual).toInclude(
-        'background-image:linear-gradient(45deg,var(--var-test-n30gray) 25%,transparent 25%),linear-gradient(-45deg,var(--var-test-n30gray) 25%,transparent 25%),linear-gradient(45deg,transparent 75%,var(--var-test-n30gray) 75%),linear-gradient(-45deg,transparent 75%,var(--var-test-n30gray) 75%)'
+        'background-image:linear-gradient(45deg,gray 25%,transparent 25%),linear-gradient(-45deg,gray 25%,transparent 25%),linear-gradient(45deg,transparent 75%,gray 75%),linear-gradient(-45deg,transparent 75%,gray 75%)'
```

Comment:
can you make another test that uses dynamic values pls

Reply:
Yep, I've added them now. Sorry, I totally forgot removing dynamic cases would remove tests that protect these features :)